actors/account/tests/account_actor_test.rs

```diff
@@ -70,14 +75,18 @@ fn token_receiver() {
     let param = Address::new_secp256k1(&[2; fvm_shared::address::SECP_PUB_LEN]).unwrap();
     rt.call::<AccountActor>(
         Method::Constructor as MethodNum,
-        &RawBytes::serialize(&param).unwrap(),
+        IpldBlock::serialize_cbor(&param).unwrap(),
     )
     .unwrap();
 
     rt.expect_validate_caller_any();
     let ret = rt.call::<AccountActor>(
         Method::UniversalReceiverHook as MethodNum,
-        &RawBytes::new(vec![1, 2, 3]),
+        IpldBlock::serialize_cbor(&UniversalReceiverParams {
```

Comment:
This is slightly different behavior, can you not just set the IPLD blocks to  bytes 1,2,3?

Reply:
(The idea of this test is that invalid bytes don't cause failureº

Reply:
The behaviour is changing so that they do cause failure. This is noted in the PR description